odoo/sql_db.py

```diff
@@ -245,9 +256,22 @@ def execute(self, query, params=None, log_exceptions=None):
             encoding = psycopg2.extensions.encodings[self.connection.encoding]
             _logger.debug("query: %s", self._obj.mogrify(query, params).decode(encoding, 'replace'))
         now = time.time()
+        catch_traceback_query = bool(os.environ.get('catch_traceback_query', ''))
```

Comment:
I think you don't need `bool` since that the `if` convert the value in bool directly